### PR TITLE
Feat: havah wallet integration

### PR DESCRIPTION
**Pull request by weeddotv**

### CODE REVIEW

**Comment by FezBox** [comment]
- [ ] Please update so hana is also supported as a havah wallet.

File: apps/web/src/app/_xcall/havah/HavahProvider/index.tsx [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+  signature: string;
+}
+interface SendTransactionResultType {
+  txHash: string;
+  type: string; // success, close, cancel, fail
+}
+
+interface HavahContextType {
+  address: string;
+  chain_id: string;
```
Comment (0xmilktea) [inline apps/web/src/app/_xcall/havah/HavahProvider/index.tsx:26]: use camelCase

File: packages/balanced-js/src/contracts/contract.ts [hunk 2]
```diff
@@ -3,7 +3,7 @@ import { isEmpty } from 'lodash-es';
 import { ICONEX_RELAY_RESPONSE } from '../iconex';
 
 import { AccountType, ResponseJsonRPCPayload, SettingInjection } from '..';
-import { SupportedChainId as NetworkId } from '../chain';
+import { SupportedChainId as NetworkId, SupportedChainId } from '../chain';
```
Comment (0xmilktea) [inline packages/balanced-js/src/contracts/contract.ts:6]: use one variable.

**Comment by parrot9design** [comment]
There are currently 3 variations of the Havah/HVH logo. Let's update them to use this one:

[Havah logo.zip](https://github.com/user-attachments/files/16257609/Havah.logo.zip)

File: apps/web/src/app/pages/test/page1.tsx [hunk 3]
```diff
@@ -0,0 +1,103 @@
+import React from 'react';
```
Comment (hetfly) [inline apps/web/src/app/pages/test/page1.tsx:1]: can we remove this?

**Pull request merged**